api-docs/leads.md

```diff
@@ -2,8 +2,86 @@
 
 The leads will document potiental clients sourced from CU events.
 
-Here is the list of the valid log types:
+## Get a single Lead
 
-- `leadCreated`,
-- `leadUpdated`,
-- `leadDeleted`
+### Request
+
+```http
+GET /api/leads/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "active",
+  "contactStage": {
+    "first": "2019-05-06T06:00:00.000Z",
+    "second": null,
+    "third": null
+  },
+  "inactivityReason": null,
+  "eventSource": {
+    "eventId": 1,
+    "eventName": "Health Fair",
+    "eventDate": "2019-09-16",
+    "eventLocation": "Saint Marks",
+    "relatedProgram": {
```

Comment:
Hmm I don't understand the `relatedProgram` thing. It isn't on https://docs.google.com/spreadsheets/d/1wGx9pu2Udnn1MEWhas3u-PTe8uwZeStj20ItJ0UDirA/edit#gid=0 as far as I can tell and isn't something we've talking about.